Source/Scene/Expression.js

```diff
@@ -226,6 +281,33 @@ define([
         return this._value;
     };
 
+    Node.prototype._evaluateVariableString = function(feature) {
+        var exp = this._value;
+        var result = "";
+        var i = exp.indexOf('${');
```

Comment:
Here's one approach using regex (I haven't actually tested it yet but it should work).  I think this does what you want. I'm sure it can be further optimized and cleaned up.  It also doesn't check for `\$` yet, but that should be doable.

@ggetz I'll stop by to walk you through this.

```javascript
var variablesRegex = /\${(.*?)}/g;

var result = expression;
var matches = variablesRegex.exec(result);
while(definedNotNull(matches)) {
	var placeholder = matches[0];
	var variableName = matches[1];
	var property = feature.getProperty(variableName);
	result = result.replace(placeholder, property);
	match = variablesRegex.match(result);
}
```